app/src/main/java/com/concordia/mcga/helperClasses/DatabaseHelper.java

```diff
@@ -5,22 +5,22 @@
 import android.database.sqlite.SQLiteException;
 import android.database.sqlite.SQLiteOpenHelper;
 import android.util.Log;
-
+import com.concordia.mcga.activities.BuildConfig;
+import com.concordia.mcga.exceptions.MCGADatabaseException;
 import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
-import java.sql.SQLException;
 
 
 //Part of the code was taken from https://blog.reigndesign.com/blog/using-your-own-sqlite-database-in-android-applications/
 public class DatabaseHelper extends SQLiteOpenHelper {
```

Comment:
How will it be a god class? Isn't the utility of the public class "DatabaseHelper" purely axed on our project?